frontend/site-registry/src/app/features/details/snapshot/SnapshotSlice.ts

```diff
@@ -98,6 +118,17 @@ const snapshotsSlice = createSlice({
       .addCase(createSnapshotForSites.rejected, (state, action) => {
         console.log('error', action);
         state.createSnapshotRequest = RequestStatus.failed;
+      })
+      .addCase(getBannerType.pending, (state) => {
+        state.status = RequestStatus.loading;
+      })
+      .addCase(getBannerType.fulfilled, (state, action) => {
+        state.status = RequestStatus.success;
```

Comment:
Please check response code returned and set the status

Reply:
Sure, will include that